.dockstore.yml

```
@@ -0,0 +1,21 @@
+# Dockstore config version, not pipeline version
+version: 1.2
+workflows:
+  - name: NPM-sample-qc
+    subclass: nfl
+    primaryDescriptorPath: /nextflow.config
+    publish: True
+    authors:
+      - name: Justin Jeyakani
+        email: gnanakkan@gis.a-star.edu.sg
+        orcid: 0000-0002-1788-0417
+      - name: Mar Gonzalez-Porta
+        email: mar@nalagenetics.com
```

```suggestion
        email: mar@nalagenetics.com
        orcid: 0000-0002-1661-7254
```
To add the orcid of Mar

---

Mar's ORCID updated.